addon/webextension/background.js

```diff
@@ -8,10 +8,50 @@
 
 
 // constants used by particular effects
-const WORDS = "dark army disintegration data privacy internet delete".split(/\s+/);
-const XHEADERSITES = ['<all_urls>'];
-const XHEADERNAME = 'dontdeleteme';
-const XHEADERVALUE = '1057'
+const WORDS =
+`
+puppet
+one
+zero
+congo
+rewind
+fuck
+71
+fsociety
+encrypt
+decrypt
+control
+illusion
+dark
+army
+evil
+robot
+disintegration
+hack
+society
+white
+rose
+revolution
+subroutine
+backdoor
+undo
+society
+corporation
+economy
+mask
+system
+truth
+debt
+cryptocurrency
+kernel panic
```

Comment:
Is this a phrase? If so, we may need to use Array instead of splitting a string?

Reply:
Per above, not being fussy.